src/main/java/com/sequenceiq/cloudbreak/service/stack/DefaultStackService.java

```diff
@@ -158,6 +157,8 @@ public void delete(Long id, CbUser user) {
             throw new NotFoundException(String.format("Stack '%s' not found", id));
         }
         delete(stack, user);
+        LOGGER.info("Publishing {} event.", ReactorConfig.DELETE_REQUEST_EVENT);
+        reactor.notify(ReactorConfig.DELETE_REQUEST_EVENT, Event.wrap(new StackDeleteRequest(stack.cloudPlatform(), stack.getId())));
```

Comment:
The delete private method already contains the code the publish a delete request event.